features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/PrivacyHelper.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal
+
+import com.datadog.android.api.InternalLogger
+
+internal class PrivacyHelper(private val internalLogger: InternalLogger) {
```

Comment:
Potentially this class could hold additional common logic in the future, and I don't see a better place at the moment to put this. Let's discuss

Reply:
I think instead of creating one class and one instance for this I would rather have this call duplicated in those classes. I am waiting to see what the others are saying also.